services/ui-src/src/components/export/ExportedModalOverlayReportSection.tsx

```diff
@@ -114,13 +134,38 @@ export function renderModalOverlayTableBody(
           </Tr>
         );
       });
-    case ReportType.MCPAR:
     case ReportType.NAAAR:
-      throw new Error(
-        `The modal overlay table headers for report type '${reportType}' have not been implemented.`
-      );
+      // render pattern for NAAAR Standards
+      return entities.map((entity, idx) => {
+        const {
+          provider,
+          standardType,
+          description,
+          analysisMethods,
+          population,
+          region,
+        } = mapNaaarStandardsData<NaaarStandardsTableShape>([entity])[0];
```

Comment:
Is it ever possible that this [0] can not be there? Double checking cause that'll hard error the user out.

Reply:
Hmm good callout 🤔 

since we are within `entities.map()` I believe entity must be defined, and in that case this function will return an array with an object of the relevant info. I also think the entity (Standards) will have to be complete because it's a drawer form, which can't submit without all fields.

I think if `entity` here were undefined or incomplete it might be a problem with the report as whole.